scss/bootstrap/variants/_checkbox-switch.scss

```diff
@@ -0,0 +1,114 @@
+.checkbox-switch {
+  padding-top: 0 !important;
+  position: relative;
+
+  input {
+    bottom: 0;
+    height: 0;
+    left: 0;
+    margin: 0;
+    right: 0;
+    top: 0;
+    visibility: hidden;
+    width: 0;
+  }
+
+  :checked {
```

Comment:
sorry this looks too weird, it should clearly be `input:checked`. Use `/* stylelint-disable-line <rule>*/`

Reply:
Nested instead